packages/Card/src/components/Footer/Footer.js

```diff
@@ -1,22 +1,50 @@
 import React from "react";
 import PropTypes from "prop-types";
+import Tag from "@paprika/tag";
 import * as sc from "./Footer.styles";
+import * as types from "./types";
 
 const propTypes = {
   /** Body content of the footer */
   children: PropTypes.node,
+
+  /** Content of the tag */
+  tagContent: PropTypes.string,
+
+  /** Themes of the tag */
+  tagTheme: PropTypes.oneOf([
+    types.themes.BLACK,
+    types.themes.BLUE,
+    types.themes.GREEN,
+    types.themes.GREY,
+    types.themes.LIGHT_BLUE,
+    types.themes.LIGHT_ORANGE,
+    types.themes.ORANGE,
+    types.severityThemes.NO_RISK,
+    types.severityThemes.LOW_RISK,
+    types.severityThemes.MEDIUM_RISK,
+    types.severityThemes.HIGH_RISK,
+    types.severityThemes.ALERT,
+  ]),
 };
 
 const defaultProps = {
   children: null,
+  tagContent: null,
+  tagTheme: types.themes.BLACK,
 };
 
 const Footer = props => {
-  const { children } = props;
+  const { children, tagContent, tagTheme } = props;
 
   return (
     <sc.footerStyles data-pka-anchor="card.footer" {...props}>
       {children}
+      {tagContent ? (
+        <Tag theme={tagTheme} size={types.sizes.MEDIUM}>
```

Comment:
This is a pretty inflexible implementation for enforcing the `MEDIUM` size of the `<Tag>`.  I actually think that this requirement, "If a tag is used, it is always a medium size 24px height", should not be the responsibility of this component at all, but a design best practice that starts with designers and is the responsibility of the app developer.  

I suggest reverting these changes to the `<Card.Footer>` and making sure we add some examples in the stories to capture the best practice for applying a `<Tag>` (with  `size={Tag.types.sizes.MEDIUM}`) in the footer.